Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php

```diff
@@ -134,7 +134,14 @@ public function getPlaceholderValues(int $userId, int $objId) : array
         $placeholder['USER_LASTNAME']      = $this->utilHelper->prepareFormOutput((trim($user->getLastname())));
         $placeholder['USER_TITLE']         = $this->utilHelper->prepareFormOutput((trim($user->getUTitle())));
         $placeholder['USER_SALUTATION']    = $this->utilHelper->prepareFormOutput($this->language->txt("salutation_" . trim($user->getGender())));
-        $placeholder['USER_BIRTHDAY']      = $this->utilHelper->prepareFormOutput((trim($user->getBirthday())));
+
+        $birthday = '';
```

Comment:
@legionth Does this really belong to this PR?

Reply:
Hm this should already have been in trunk, I will check this

Reply:
Thx

Reply:
Fixed via rebased. Ready for Review!